src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+body {
```

Comment:
reed about [universal selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors)
using this selector you can delete all margin: 0 below it